xun/functions/store/sftp.py

```diff
@@ -84,11 +84,10 @@ def ssh(self):
     def sftp(self):
         if self._sftp is None:
             self._sftp = self.ssh.open_sftp()
-            self._sftp.chdir(str(self.root))
             if not self.is_dir(self.root / 'values'):
-                self._sftp.mkdir('values', mode=0o711)
+                self._sftp.mkdir(str(self.root / 'values'), mode=0o711)
             if not self.is_dir(self.root / 'keys'):
-                self._sftp.mkdir('keys', mode=0o711)
+                self._sftp.mkdir(str(self.root / 'keys'), mode=0o711)
```

Comment:
This seems wrong? 🤔

Reply:
It's the result of Paramiko nor really supporting Win OS in general. It only checks if working directory is already set in unix-based paths. Which means it checks if path starts with '/'. That is not present in Win based paths. In short by setting chdir first and then calling mkdir, on windows size the path created is something like: 'C:\...C:\....\values'.

This way we are on safe ground on multiple OS

Reply:
This is fine